types/index.d.ts

```diff
@@ -136,6 +136,17 @@ declare namespace jest {
      */
     toHaveBeenCalledAfter(mock: jest.Mock): R;
 
+    /**
+     * Use `.toHaveBeenCalledOnce` when checking if a `Mock` was called exactly once.
+     */
+    toHaveBeenCalledOnce(): R;
+
+    /**
+     * Use `.toHaveBeenCalledOnceWith` when checking if a `Mock` was called once with expected values.
+     * @param {Array.<*>} expected
+     */
+    toHaveBeenCalledOnceWith(expected: any[]): R;
```

Comment:
Is this the right type for spread arguments?

Reply:
as per my comment in aha missed this :P, as per my comment over in the toReject/toResolve PR I would appreciate any resources you have to understand the typescript definitions better :)